src/Nethermind/Nethermind.Network.Discovery.Test/Nethermind.Network.Discovery.Test.csproj

```diff
@@ -8,8 +8,8 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="DotNetty.Transport" Version="0.7.2" />
         <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
+        <PackageReference Include="Nethermind.DotNetty.Transport" Version="0.7.5-preview.1" />
```

Comment:
why preview?

Reply:
Cause we're testing it.

Reply:
Don't merge it to master then - this should be a draft

Reply:
Just to clarify. I'm done with this PR and marked it as ready. It's now up to you what to do next, keep it a draft, merge, etc.

Reply:
Lets run smoke tests + local nodes and if everything is fine mark it as stable and update this PR?

Reply:
Alright